src/h5web/toolbar/Toolbar.module.css

```diff
@@ -64,8 +66,7 @@
 }
 
 .btn[data-raised] > .btnLike {
-  background-color: var(--btn-bg-raised);
```

Comment:
"raised" buttons are used for min/max auto-scaling in the domain slider's tooltip. I give them a slightly bigger shadow and remove their background, which allows me to use them in toggle groups to avoid having another button variant (which would mean more CSS variables).